output-blocks/src/Control/OutputCapable/Blocks.hs

```diff
@@ -501,6 +503,19 @@ You should always prefer this specified version over the generic.
 translations :: State (Map l a) () -> Map l a
 translations = Generic.translations
 
+{-|
+This is a specified version of 'Generic.collapsed'
+which enforces the usage pattern.
+You should always prefer this specified version over the generic.
```

Comment:
The usage of "specified" here is strange. (I know it is copied from elsewhere. So there should be a subsequent PR fixing it everywhere.)

Suggestion (everywhere): "specified" -> "more specific".